### ISSUE TITLE
minikube start with version 1.12.1 fails: waiting for apiserver: apiserver process never appeared

### ISSUE THREAD

**Issue by RoelandMatthijssens** [kubernetes/minikube#6026]
**The exact command to reproduce the issue**: 
`minikube start --kubernetes-version v1.12.1`

**The full output of the command that failed**:<details>

```
 ~> minikube delete
🔥  Deleting "minikube" in hyperkit ...
💔  The "minikube" cluster has been deleted.
🔥  Successfully deleted profile "minikube"
 ✘ ~> minikube start --kubernetes-version v1.7.15
😄  minikube v1.5.2 on Darwin 10.15.1
✨  Automatically selected the 'hyperkit' driver
🔥  Creating hyperkit VM (CPUs=2, Memory=2000MB, Disk=20000MB) ...
🐳  Preparing Kubernetes v1.7.15 on Docker '18.09.9' ...
💾  Downloading kubelet v1.7.15
💾  Downloading kubeadm v1.7.15
^C
✘ ~> minikube start --kubernetes-version v1.12.1
😄  minikube v1.5.2 on Darwin 10.15.1
👍  Upgrading from Kubernetes 1.7.15 to 1.12.1
💡  Tip: Use 'minikube start -p <name>' to create a new cluster, or 'minikube delete' to delete this one.
🔄  Starting existing hyperkit VM for "minikube" ...
⌛  Waiting for the host to be provisioned ...
🐳  Preparing Kubernetes v1.12.1 on Docker '18.09.9' ...
💾  Downloading kubeadm v1.12.1
💾  Downloading kubelet v1.12.1
🚜  Pulling images ...
🔄  Relaunching Kubernetes using kubeadm ...

💣  Error restarting cluster: waiting for apiserver: apiserver process never appeared

😿  Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
👉  https://github.com/kubernetes/minikube/issues/new/choose
✘ ~>  minikube start --kubernetes-version v1.12.1
😄  minikube v1.5.2 on Darwin 10.15.1
💡  Tip: Use 'minikube start -p <name>' to create a new cluster, or 'minikube delete' to delete this one.
🏃  Using the running hyperkit "minikube" VM ...
⌛  Waiting for the host to be provisioned ...
🐳  Preparing Kubernetes v1.12.1 on Docker '18.09.9' ...
🔄  Relaunching Kubernetes using kubeadm ...

💣  Error restarting cluster: waiting for apiserver: apiserver process never appeared

😿  Sorry that minikube crashed. If this was unexpected, we would love to hear from you:
👉  https://github.com/kubernetes/minikube/issues/new/choose
✘ ~> 
```
</details>

**The output of the `minikube logs` command**:<details>
==> Docker <==
-- Logs begin at Fri 2019-12-06 16:02:03 UTC, end at Fri 2019-12-06 16:16:25 UTC. --
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108066709Z" level=info msg="loading plugin "io.containerd.gc.v1.scheduler"..." type=io.containerd.gc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108221630Z" level=info msg="loading plugin "io.containerd.service.v1.containers-service"..." type=io.containerd.service.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108383959Z" level=info msg="loading plugin "io.containerd.service.v1.content-service"..." type=io.containerd.service.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108445675Z" level=info msg="loading plugin "io.containerd.service.v1.diff-service"..." type=io.containerd.service.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108586557Z" level=info msg="loading plugin "io.containerd.service.v1.images-service"..." type=io.containerd.service.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108668526Z" level=info msg="loading plugin "io.containerd.service.v1.leases-service"..." type=io.containerd.service.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108732879Z" level=info msg="loading plugin "io.containerd.service.v1.namespaces-service"..." type=io.containerd.service.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108798874Z" level=info msg="loading plugin "io.containerd.service.v1.snapshots-service"..." type=io.containerd.service.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.108847369Z" level=info msg="loading plugin "io.containerd.runtime.v1.linux"..." type=io.containerd.runtime.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.109105004Z" level=info msg="loading plugin "io.containerd.runtime.v2.task"..." type=io.containerd.runtime.v2
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.109246336Z" level=info msg="loading plugin "io.containerd.monitor.v1.cgroups"..." type=io.containerd.monitor.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.109911269Z" level=info msg="loading plugin "io.containerd.service.v1.tasks-service"..." type=io.containerd.service.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110060559Z" level=info msg="loading plugin "io.containerd.internal.v1.restart"..." type=io.containerd.internal.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110216377Z" level=info msg="loading plugin "io.containerd.grpc.v1.containers"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110290266Z" level=info msg="loading plugin "io.containerd.grpc.v1.content"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110334452Z" level=info msg="loading plugin "io.containerd.grpc.v1.diff"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110397653Z" level=info msg="loading plugin "io.containerd.grpc.v1.events"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110483836Z" level=info msg="loading plugin "io.containerd.grpc.v1.healthcheck"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110594888Z" level=info msg="loading plugin "io.containerd.grpc.v1.images"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110687482Z" level=info msg="loading plugin "io.containerd.grpc.v1.leases"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110792344Z" level=info msg="loading plugin "io.containerd.grpc.v1.namespaces"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.110889828Z" level=info msg="loading plugin "io.containerd.internal.v1.opt"..." type=io.containerd.internal.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.111019615Z" level=info msg="loading plugin "io.containerd.grpc.v1.snapshots"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.111146988Z" level=info msg="loading plugin "io.containerd.grpc.v1.tasks"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.111215026Z" level=info msg="loading plugin "io.containerd.grpc.v1.version"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.111280826Z" level=info msg="loading plugin "io.containerd.grpc.v1.introspection"..." type=io.containerd.grpc.v1
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.111509868Z" level=info msg=serving... address="/var/run/docker/containerd/containerd-debug.sock"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.111633509Z" level=info msg=serving... address="/var/run/docker/containerd/containerd.sock"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.111698868Z" level=info msg="containerd successfully booted in 0.011204s"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.116576146Z" level=info msg="pickfirstBalancer: HandleSubConnStateChange: 0xc000163970, READY" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.122484720Z" level=info msg="systemd-resolved is running, so using resolvconf: /run/systemd/resolve/resolv.conf"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.123563185Z" level=info msg="parsed scheme: \"unix\"" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.123732807Z" level=info msg="scheme \"unix\" not registered, fallback to default scheme" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.123951232Z" level=info msg="parsed scheme: \"unix\"" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.124065228Z" level=info msg="scheme \"unix\" not registered, fallback to default scheme" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.123956715Z" level=info msg="ccResolverWrapper: sending new addresses to cc: [{unix:///var/run/docker/containerd/containerd.sock 0  <nil>}]" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.124331943Z" level=info msg="ClientConn switching balancer to \"pick_first\"" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.124377332Z" level=info msg="pickfirstBalancer: HandleSubConnStateChange: 0xc000163dd0, CONNECTING" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.124800650Z" level=info msg="pickfirstBalancer: HandleSubConnStateChange: 0xc000163dd0, READY" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.124964193Z" level=info msg="ccResolverWrapper: sending new addresses to cc: [{unix:///var/run/docker/containerd/containerd.sock 0  <nil>}]" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.125018825Z" level=info msg="ClientConn switching balancer to \"pick_first\"" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.125083415Z" level=info msg="pickfirstBalancer: HandleSubConnStateChange: 0xc0006fe0b0, CONNECTING" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.125514502Z" level=info msg="pickfirstBalancer: HandleSubConnStateChange: 0xc0006fe0b0, READY" module=grpc
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.127502834Z" level=info msg="[graphdriver] using prior storage driver: overlay2"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.160727165Z" level=info msg="Graph migration to content-addressability took 0.00 seconds"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.161327169Z" level=warning msg="Your kernel does not support cgroup blkio weight"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.161434964Z" level=warning msg="Your kernel does not support cgroup blkio weight_device"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.161540999Z" level=warning msg="Your kernel does not support cgroup blkio throttle.read_bps_device"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.161606583Z" level=warning msg="Your kernel does not support cgroup blkio throttle.write_bps_device"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.161668663Z" level=warning msg="Your kernel does not support cgroup blkio throttle.read_iops_device"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.161734809Z" level=warning msg="Your kernel does not support cgroup blkio throttle.write_iops_device"
Dec 06 16:02:16 minikube dockerd[1908]: time="2019-12-06T16:02:16.162485030Z" level=info msg="Loading containers: start."
Dec 06 16:02:17 minikube dockerd[1908]: time="2019-12-06T16:02:17.163819273Z" level=info msg="Default bridge (docker0) is assigned with an IP address 172.17.0.0/16. Daemon option --bip can be used to set a preferred IP address"
Dec 06 16:02:17 minikube dockerd[1908]: time="2019-12-06T16:02:17.214050133Z" level=info msg="Loading containers: done."
Dec 06 16:02:17 minikube dockerd[1908]: time="2019-12-06T16:02:17.224616512Z" level=warning msg="failed to retrieve runc version: unknown output format: runc version commit: 425e105d5a03fabd737a126ad93d62a9eeede87f\nspec: 1.0.1-dev\n"
Dec 06 16:02:17 minikube dockerd[1908]: time="2019-12-06T16:02:17.236322699Z" level=info msg="Docker daemon" commit=039a7df9ba graphdriver(s)=overlay2 version=18.09.9
Dec 06 16:02:17 minikube dockerd[1908]: time="2019-12-06T16:02:17.236591350Z" level=info msg="Daemon has completed initialization"
Dec 06 16:02:17 minikube dockerd[1908]: time="2019-12-06T16:02:17.245968762Z" level=info msg="API listen on /var/run/docker.sock"
Dec 06 16:02:17 minikube systemd[1]: Started Docker Application Container Engine.
Dec 06 16:02:17 minikube dockerd[1908]: time="2019-12-06T16:02:17.247908014Z" level=info msg="API listen on [::]:2376"

==> container status <==
time="2019-12-06T16:16:27Z" level=fatal msg="failed to connect: failed to connect, make sure you are running as root and the runtime has been started: context deadline exceeded"
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES

==> dmesg <==
[Dec 6 16:01] ERROR: earlyprintk= earlyser already used
[  +0.000000] You have booted with nomodeset. This means your GPU drivers are DISABLED
[  +0.000000] Any video related functionality will be severely degraded, and you may not even be able to suspend the system properly
[  +0.000001] Unless you actually understand what nomodeset does, you should reboot without enabling it
[  +0.010733] ACPI BIOS Warning (bug): Incorrect checksum in table [DSDT] - 0xC0, should be 0x1D (20180810/tbprint-177)
[Dec 6 16:02] ACPI Error: Could not enable RealTimeClock event (20180810/evxfevnt-184)
[  +0.000003] ACPI Warning: Could not enable fixed event - RealTimeClock (4) (20180810/evxface-620)
[  +0.006942] platform regulatory.0: Direct firmware load for regulatory.db failed with error -2
[  +2.022793] systemd[1]: Failed to bump fs.file-max, ignoring: Invalid argument
[  +0.006637] systemd-fstab-generator[1083]: Ignoring "noauto" for root device
[  +0.003271] systemd[1]: File /usr/lib/systemd/system/systemd-journald.service:12 configures an IP firewall (IPAddressDeny=any), but the local system does not support BPF/cgroup based firewalling.
[  +0.000001] systemd[1]: Proceeding WITHOUT firewalling in effect! (This warning is only shown for the first loaded unit using IP firewalling.)
[  +1.316801] vboxguest: loading out-of-tree module taints kernel.
[  +0.002685] vboxguest: PCI device not found, probably running on physical hardware.
[  +0.086395] NFSD: the nfsdcld client tracking upcall will be removed in 3.10. Please transition to using nfsdcltrack.
[ +10.682594] systemd-fstab-generator[1838]: Ignoring "noauto" for root device
[ +34.817950] systemd-fstab-generator[2461]: Ignoring "noauto" for root device
[Dec 6 16:04] NFSD: Unable to end grace period: -110
[Dec 6 16:07] systemd-fstab-generator[6415]: Ignoring "noauto" for root device
[  +7.062013] systemd-fstab-generator[6749]: Ignoring "noauto" for root device

==> kernel <==
 16:16:27 up 14 min,  0 users,  load average: 0.15, 0.23, 0.22
Linux minikube 4.19.76 #1 SMP Tue Oct 29 14:56:42 PDT 2019 x86_64 GNU/Linux
PRETTY_NAME="Buildroot 2019.02.6"

==> kubelet <==
-- Logs begin at Fri 2019-12-06 16:02:03 UTC, end at Fri 2019-12-06 16:16:27 UTC. --
Dec 06 16:16:24 minikube systemd[1]: kubelet.service: Service RestartSec=600ms expired, scheduling restart.
Dec 06 16:16:24 minikube systemd[1]: kubelet.service: Scheduled restart job, restart counter is at 1059.
Dec 06 16:16:24 minikube systemd[1]: Stopped kubelet: The Kubernetes Node Agent.
Dec 06 16:16:24 minikube systemd[1]: Started kubelet: The Kubernetes Node Agent.
Dec 06 16:16:24 minikube kubelet[14161]: Flag --allow-privileged has been deprecated, will be removed in a future version
Dec 06 16:16:24 minikube kubelet[14161]: Flag --authorization-mode has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:24 minikube kubelet[14161]: Flag --cgroup-driver has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:24 minikube kubelet[14161]: Flag --client-ca-file has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:24 minikube kubelet[14161]: Flag --cluster-dns has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:24 minikube kubelet[14161]: Flag --cluster-domain has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:24 minikube kubelet[14161]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:24 minikube kubelet[14161]: Flag --pod-manifest-path has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:24 minikube kubelet[14161]: F1206 16:16:24.677268   14161 server.go:190] failed to load Kubelet config file /var/lib/kubelet/config.yaml, error failed to read kubelet config file "/var/lib/kubelet/config.yaml", error: open /var/lib/kubelet/config.yaml: no such file or directory
Dec 06 16:16:24 minikube systemd[1]: kubelet.service: Main process exited, code=exited, status=255/EXCEPTION
Dec 06 16:16:24 minikube systemd[1]: kubelet.service: Failed with result 'exit-code'.
Dec 06 16:16:25 minikube systemd[1]: kubelet.service: Service RestartSec=600ms expired, scheduling restart.
Dec 06 16:16:25 minikube systemd[1]: kubelet.service: Scheduled restart job, restart counter is at 1060.
Dec 06 16:16:25 minikube systemd[1]: Stopped kubelet: The Kubernetes Node Agent.
Dec 06 16:16:25 minikube systemd[1]: Started kubelet: The Kubernetes Node Agent.
Dec 06 16:16:25 minikube kubelet[14234]: Flag --allow-privileged has been deprecated, will be removed in a future version
Dec 06 16:16:25 minikube kubelet[14234]: Flag --authorization-mode has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:25 minikube kubelet[14234]: Flag --cgroup-driver has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:25 minikube kubelet[14234]: Flag --client-ca-file has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:25 minikube kubelet[14234]: Flag --cluster-dns has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:25 minikube kubelet[14234]: Flag --cluster-domain has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:25 minikube kubelet[14234]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:25 minikube kubelet[14234]: Flag --pod-manifest-path has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:25 minikube kubelet[14234]: F1206 16:16:25.412956   14234 server.go:190] failed to load Kubelet config file /var/lib/kubelet/config.yaml, error failed to read kubelet config file "/var/lib/kubelet/config.yaml", error: open /var/lib/kubelet/config.yaml: no such file or directory
Dec 06 16:16:25 minikube systemd[1]: kubelet.service: Main process exited, code=exited, status=255/EXCEPTION
Dec 06 16:16:25 minikube systemd[1]: kubelet.service: Failed with result 'exit-code'.
Dec 06 16:16:26 minikube systemd[1]: kubelet.service: Service RestartSec=600ms expired, scheduling restart.
Dec 06 16:16:26 minikube systemd[1]: kubelet.service: Scheduled restart job, restart counter is at 1061.
Dec 06 16:16:26 minikube systemd[1]: Stopped kubelet: The Kubernetes Node Agent.
Dec 06 16:16:26 minikube systemd[1]: Started kubelet: The Kubernetes Node Agent.
Dec 06 16:16:26 minikube kubelet[14243]: Flag --allow-privileged has been deprecated, will be removed in a future version
Dec 06 16:16:26 minikube kubelet[14243]: Flag --authorization-mode has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14243]: Flag --cgroup-driver has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14243]: Flag --client-ca-file has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14243]: Flag --cluster-dns has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14243]: Flag --cluster-domain has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14243]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14243]: Flag --pod-manifest-path has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14243]: F1206 16:16:26.165342   14243 server.go:190] failed to load Kubelet config file /var/lib/kubelet/config.yaml, error failed to read kubelet config file "/var/lib/kubelet/config.yaml", error: open /var/lib/kubelet/config.yaml: no such file or directory
Dec 06 16:16:26 minikube systemd[1]: kubelet.service: Main process exited, code=exited, status=255/EXCEPTION
Dec 06 16:16:26 minikube systemd[1]: kubelet.service: Failed with result 'exit-code'.
Dec 06 16:16:26 minikube systemd[1]: kubelet.service: Service RestartSec=600ms expired, scheduling restart.
Dec 06 16:16:26 minikube systemd[1]: kubelet.service: Scheduled restart job, restart counter is at 1062.
Dec 06 16:16:26 minikube systemd[1]: Stopped kubelet: The Kubernetes Node Agent.
Dec 06 16:16:26 minikube systemd[1]: Started kubelet: The Kubernetes Node Agent.
Dec 06 16:16:26 minikube kubelet[14252]: Flag --allow-privileged has been deprecated, will be removed in a future version
Dec 06 16:16:26 minikube kubelet[14252]: Flag --authorization-mode has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14252]: Flag --cgroup-driver has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14252]: Flag --client-ca-file has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14252]: Flag --cluster-dns has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14252]: Flag --cluster-domain has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14252]: Flag --fail-swap-on has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14252]: Flag --pod-manifest-path has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Dec 06 16:16:26 minikube kubelet[14252]: F1206 16:16:26.914969   14252 server.go:190] failed to load Kubelet config file /var/lib/kubelet/config.yaml, error failed to read kubelet config file "/var/lib/kubelet/config.yaml", error: open /var/lib/kubelet/config.yaml: no such file or directory
Dec 06 16:16:26 minikube systemd[1]: kubelet.service: Main process exited, code=exited, status=255/EXCEPTION
Dec 06 16:16:26 minikube systemd[1]: kubelet.service: Failed with result 'exit-code'.
</details>

**The operating system version**: macOS Catalina 10.15.1 (19B88)

**Comment by tstromberg** [comment]
I believe we fixed this with #5916 which was merged into minikube v1.6. Do you mind checking with the most recent release?

**Comment by RoelandMatthijssens** [comment]
Yup you are correct, upgrading to 1.6.1 allows me to run kubernetes v1.12.1. Thanks
